html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="GET">
+    <label for="email">Email</label>
+    <div id="email">
+      <input type="text" name="email">
+    </div>
+    <label for="why-coding">Why is coding cool?</label>
+    <div id="why-coding">
+      <select name="why-coding">
+        <option value="select-option">Select an option</option>
+        <option value="coding-interesting">Because it's interesting</option>
+        <option value="coding-fun">Because it's fun</option>
+        <option value="coding-powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <label for="favorite-language">Favorite Language (choose one)</label>
```

Comment:
This doesn't need to be a `label`, it doesn't match to one specific form control element.